requirements.txt

```diff
@@ -21,5 +21,6 @@ nbmake
 
 # optional rt/test dependencies
 tensorflow
+cuda-quantum
```

Comment:
this breaks the requirements.txt compatibility on macOS.

Reply:
Removed: https://github.com/PennyLaneAI/catalyst/pull/477/commits/ccf6619b5278de9ea01ccf814ab8ee8089f615e9